src/prospectScraper/ProspectScraper.cs

```diff
@@ -53,85 +55,63 @@ public void RunTheBigBoards(bool parseDate = true)
+                if (list4.Any())
                 {
                     csv.WriteRecords(list4);
                 }
-                if (list5.Count > 0)
+                if (list5.Any())
                 {
                     csv.WriteRecords(list5);
                 }
             }
 
             CheckForMismatches(csvFileName);
-            CreateCombinedCSV();
+            CreateCombinedCsv();
             CheckForMismatches($"ranks{Path.DirectorySeparatorChar}combinedRanks2021.csv");
-            CreateCombinedCSVWithExtras();
+            await CreateCombinedCsvWithExtras();
 
             Console.WriteLine("Big Board Completed.");
         }
 
-        public void RunTheMockDraft(bool parseDate)
+        public async Task RunTheMockDraft(bool parseDate)
         {
-            var webGet = new HtmlWeb
-            {
-                UserAgent = "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:31.0) Gecko/20100101 Firefox/31.0"
-            };
-            var document1 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1.asp");
-            var document3 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-2.asp");
-            var document4 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-3.asp");
-            var document5 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-4.asp");
-            var document6 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-6.asp");
-
-            // Need to get date of mock draft eventually.
-            HtmlNode hn = document1.DocumentNode;
-            HtmlNode hi1 = hn.SelectSingleNode("//*[@id='HeadlineInfo1']");
-            DateTime mockDraftDate = ChangeDateStringToDateTime(hi1.InnerText.Replace(" EST", "").Trim(), parseDate);
-            string draftDate = mockDraftDate.ToString("yyyy-MM-dd");
-
-
-            List<MockDraftPick> list1 = GetMockDraft(document1, draftDate);
-            List<MockDraftPick> list2 = GetMockDraft(webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1b.asp"), draftDate);
-            List<MockDraftPick> list3 = GetMockDraft(document3, draftDate);
-            List<MockDraftPick> list4 = GetMockDraft(document4, draftDate);
-            List<MockDraftPick> list5 = GetMockDraft(document5, draftDate);
-            List<MockDraftPick> list6 = GetMockDraft(document6, draftDate);
+            string draftDate = string.Empty;
 
-            //This is the file name we are going to write.
-            var csvFileName = $"mocks{Path.DirectorySeparatorChar}{draftDate}-mock.csv";
+            var listOfDraftPicks = new List<MockDraftPick>();
+            for (int i = 1; i < 7; i++)
+            {
+                var sb = new StringBuilder($"https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-");
+                sb.Append($"{i}.asp");
```

Comment:
I don't think that this will work for the Mock Draft pages.
The order for mock draft pages is:

- *Round1.asp
- *Round-1b.asp
- *Round-2.asp
- *Round-3.asp
- *Round-4.asp
- *Round-6.asp

Because of this wacky order, a normal loop isn't going to work (though it would work for the big board pages).